src/api.rs

```diff
@@ -8,18 +8,19 @@
 //! # fn main() {}
 //! # #[cfg(not(miri))]
 //! # fn main() {
-//! use nemo::api::{load,reason,write};
+//! use nemo::api::{load,reason,output_predicates,write};
```

Comment:
Please fix the formatting for the example (I realise it was already broken before, but there should be a space after each comma).